### ISSUE TITLE
Page on "CLR strict security" is unhelpful

### ISSUE THREAD

**Issue by ashtonbatty** [MicrosoftDocs/sql-docs#8765]
The page does not link to instructions on signing and deploying assemblies for where CLR strict security is enabled. The page provides just enough information and links for disabling CLR strict security (though that itself isn't very clear), but only provides hints of how to follow the recommended path of leaving strict security enabled.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 96693d4a-9460-9e08-8f14-836b6257e449
* Version Independent ID: 47f19aec-1f62-97c4-fca4-609496fc2139
* Content: [CLR strict security - SQL Server](https://learn.microsoft.com/en-us/sql/database-engine/configure-windows/clr-strict-security?source=recommendations&view=sql-server-ver16)
* Content Source: [docs/database-engine/configure-windows/clr-strict-security.md](https://github.com/MicrosoftDocs/sql-docs/blob/live/docs/database-engine/configure-windows/clr-strict-security.md)
* Service: **sql**
* Sub-service: **configuration**
* GitHub Login: @rwestMSFT
* Microsoft Alias: **randolphwest**

**Comment by rwestMSFT** [comment]
Did you see this article?

[sys.sp_add_trusted_assembly (Transact-SQL)](https://learn.microsoft.com/en-us/sql/relational-databases/system-stored-procedures/sys-sp-add-trusted-assembly-transact-sql?view=sql-server-ver16)

**Comment by ashtonbatty** [comment]
Yes, and I used sp_add_trusted_assembly for some initial development. However it is only mentioned in a 'warning' info box, and not mentioned in the main content.

The main content recommends deploying signed assemblies: "The assembly is signed with a certificate or asymmetric key that has a corresponding login with the `UNSAFE ASSEMBLY` permission on the server. **Signing the assembly is recommended.** "

It is this recommendation that is a dead end, with no further linking to pages or articles explaining how create a certificate or key, how to sign an assembly with the created certificate or key, how to deploy the public certificate or key to database, or how to create a 'corresponding login' for the certificate or key. 

Searching for relevant, coherent, and complete instructions in Microsoft documentation is not trivial either, and in the end I figured it out using Stackoverflow and blog posts, rather than from official documentation (problematic when Stackoverflow and most blog/personal websites are blocked by a customer's restrictive firewall/proxy rules).
